### PR TITLE
ROC-5033:Nightly pipeline

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5033

### Change description
changing nightly pipeline config

### CODE REVIEW

File: docker-compose.nightly-functional-tests.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ version: '2.1'
 services:
     citizen-integration-tests:
       image: hmcts/cmc-citizen-integration-tests:${CITIZEN_INTEGRATION_TESTS_VERSION:-latest}
-      command: test:integration --grep '^(?!.*@nightly)'
+      command: test:integration --grep @nightly
       environment:
         - WEB_DRIVER_HOST=remote-webdriver
-        - CITIZEN_APP_URL=${TEST_URL}
+        - CITIZEN_APP_URL=${TEST_URL}ok,
```
Comment (markry-kainos) [inline docker-compose.nightly-functional-tests.yml]: `ok` :)

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -12,6 +12,29 @@ properties([
...
+]
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
+  [ $class: 'AzureKeyVaultSecret',
+    secretType: 'Secret',
+    name: secretName,
+    version: '',
+    envVariable: envVar
+  ]
+}
+// Vars needed for AKS testing - keep values in sync with: /src/kubernetes/deployment.template.yaml
```
Comment (markry-kainos) [inline Jenkinsfile_nightly]: drop this line - not relevant here

File: Jenkinsfile_nightly [hunk 3]
```diff
@@ -12,6 +12,29 @@ properties([
...
+// THESE WILL BE OVERWRITTEN IN ASE DEPLOYMENTS WITH TERRAFORM OUTPUTS.TF
```
Comment (markry-kainos) [inline Jenkinsfile_nightly]: drop this line too

File: docker-compose.nightly-functional-tests.yml (hunk 1 above)
Comment (KiranVarma02) [reply docker-compose.nightly-functional-tests.yml]: haha that must be what i wanted to reply to you in slack :)

File: Jenkinsfile_nightly (hunk 2 above)
Comment (KiranVarma02) [reply Jenkinsfile_nightly]: removed

File: Jenkinsfile_nightly (hunk 3 above)
Comment (KiranVarma02) [reply Jenkinsfile_nightly]: removed

File: Jenkinsfile_nightly (hunk 2 above)
Comment (KiranVarma02) [reply Jenkinsfile_nightly]: done

File: Jenkinsfile_nightly (hunk 3 above)
Comment (KiranVarma02) [reply Jenkinsfile_nightly]: done

**Pull request merged**